src/app/models/propTypes/storyItem/index.js

```diff
@@ -1,10 +1,10 @@
 import { shape, string, number } from 'prop-types';
 
 export const storyItemImage = {
-  path: string.isRequired,
-  altText: string.isRequired,
-  height: number.isRequired,
-  width: number.isRequired,
+  path: string,
```

Comment:
should these not be required?

Reply:
@hindsc52 if you look at a [component using these props](https://github.com/bbc/simorgh/blob/11d855d1ecfb7065af3d113598a9fcf93af05a27/src/app/containers/StoryPromo/index.jsx#L76-L99) you can see they are actually optional

Reply:
also here https://github.com/bbc/simorgh/blob/11d855d1ecfb7065af3d113598a9fcf93af05a27/src/app/containers/StoryPromo/index.jsx#L22